cloudnet/src/main/java/de/dytanic/cloudnet/CloudNet.java

```diff
@@ -727,8 +727,10 @@ public void runCommand(ServiceInfoSnapshot serviceInfoSnapshot, String command)
 
     @Override
     public ServiceInfoSnapshot getCloudServiceByName(String name) {
-        ICloudService service = Iterables.first(cloudServiceManager.getCloudServices().values(), cloudService -> cloudService.getServiceId().getName().equalsIgnoreCase(name));
-        return service != null ? service.getServiceInfoSnapshot() : null;
+        return this.cloudServiceManager.getGlobalServiceInfoSnapshots().values().stream()
+                .filter(serviceInfoSnapshot -> serviceInfoSnapshot.getServiceId().getName().equalsIgnoreCase(name))
```

Comment:
Comparing the uuid of the service might be better here

Reply:
What do you mean with that?

Reply:
Services have uuids, use the name to compare them

Reply:
Oh bruh